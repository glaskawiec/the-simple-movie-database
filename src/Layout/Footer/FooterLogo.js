import styled from 'styled-components';

const FooterLogo = styled.div`
   font-family: 'Bebas Neue',serif;
    font-style: normal;
    font-weight: 500;
    color: #FFFFFF;
    font-size: 1.5em;
    border: solid .1em ${props => props.theme.colors.primary};
    border-bottom: 0;
    padding: .15em;
    display: inline-block;
    text-transform: uppercase;
`;

export default FooterLogo;
