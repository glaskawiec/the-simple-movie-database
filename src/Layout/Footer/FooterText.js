import styled from 'styled-components';

const FooterText = styled.div`
   font-family: 'Karla',serif;
    font-style: normal;
    font-weight: 500;
    color: #FFFFFF;
    margin-left: 1em;
`;

export default FooterText;
