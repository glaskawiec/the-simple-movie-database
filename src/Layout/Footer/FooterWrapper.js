import styled from 'styled-components';

const FooterWrapper = styled.div`
  background-color: #161616;
  width: 100vw;
  height: 4em;
  display: flex;
  align-items: center;
  justify-content: center;
`;

export default FooterWrapper;
