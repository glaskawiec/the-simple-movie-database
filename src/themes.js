const Themes = {
  light: {
    colors: {
      primary: '#F2355B',
      secondary: '#161616',
      background: '#FFFFFF',
      text: '#000000',
      mobileMenuOverlay: '#000000',
      card: '#FFFFFF',
      themeToggleThumb: '#161616',
      paginationHover: '#CDCDCD',
    },
  },
  dark: {
    colors: {
      primary: '#F2355B',
      secondary: '#090909',
      background: '#161616',
      text: 'hsla(0,0%,100%,0.88)',
      mobileMenuOverlay: '#FFFFFF',
      card: '#090909',
      themeToggleThumb: '#FFFFFF',
      paginationHover: '#292929',
    },
  },
};

export default Themes;
