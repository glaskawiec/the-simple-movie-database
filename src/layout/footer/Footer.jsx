import React from 'react';
import FooterWrapper from './FooterWrapper';
import FooterText from './FooterText';
import FooterLogo from './FooterLogo';

const Footer = () => (
  <FooterWrapper>
    <FooterLogo>tsmdb</FooterLogo>
    <FooterText>
      {'© 2019 by Gilbert Łaskawiec.'}
    </FooterText>
  </FooterWrapper>
);

export default Footer;
