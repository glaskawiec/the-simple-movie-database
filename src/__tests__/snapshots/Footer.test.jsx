import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import themes from '../../themes';
import Footer from '../../layout/footer/Footer';

test('Footer snapshot', () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={themes.light}>
        <Footer />
      </ThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
