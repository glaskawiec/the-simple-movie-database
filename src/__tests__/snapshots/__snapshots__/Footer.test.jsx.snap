// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<footer
  className="sc-bdVaJa jYspwc"
>
  <div
    className="sc-htpNat bzegzi"
  >
    tsmdb
  </div>
  <div
    className="sc-bwzfXH eygQfQ"
  >
    © 2019 by Gilbert Łaskawiec.
  </div>
</footer>
`;
